workflows/python/sdk/README.md

```diff
@@ -0,0 +1,107 @@
+- update_inventory_activity: This activity removes the requested items from the state store and updates the store with the new remaining inventory value.
+- requst_approval_activity: This activity seeks approval from Manager, if payment is greater than 50000 USD.
+
+### Run the order processor workflow
+
+1. Open a new terminal window and navigate to `order-processor` directory: 
+
+<!-- STEP
+name: Install requirements
+-->
+
+```sh
+cd ./order-processor
+pip3 install -r requirements.txt
+```
+
+<!-- END_STEP -->
+2. Run the console app with Dapr: 
+<!-- STEP
+name: Running this example
+expected_stdout_lines:
+  - "There are now 89 cars left in stock"
+  - "Purchase of item is  COMPLETED"
+output_match_mode: substring
+background: true
+timeout_seconds: 30
+sleep: 15
+-->
+
+```sh
+cd ./order-processor
+dapr run --app-id order-processor --app-protocol grpc --dapr-grpc-port 4001 --components-path ../../../components/ --placement-host-address localhost:50005 -- python3 app.py
+```
+
+<!-- END_STEP -->
+
+3. Expected output
+
+```
+==========Begin the purchase of item:==========
+To restock items, type 'restock'.
+To exit workflow console app, type 'exit'.
+Enter the name of one of the following items to order: paperclip, cars, computers: cars
+How many cars would you like to purchase? 11
+Starting order workflow, purchasing 11 of cars
+INFO:NotifyActivity:Received order b903d749cd814e099f06ebf4a56a2f90 for 11 cars at $165000 !
+INFO:VerifyInventoryActivity:Verifying inventory for order b903d749cd814e099f06ebf4a56a2f90 of 11 cars
+INFO:VerifyInventoryActivity:There are 100 Cars available for purchase
+INFO:RequestApprovalActivity:Requesting approval for payment of 165000 USD for 11 cars
+(ID = b903d749cd814e099f06ebf4a56a2f90) requires approval. Approve? [Y/N] y
+INFO:NotifyActivity:Payment for order b903d749cd814e099f06ebf4a56a2f90 has been approved!
+INFO:ProcessPaymentActivity:Processing payment: b903d749cd814e099f06ebf4a56a2f90 for 11 cars at 165000 USD
+INFO:ProcessPaymentActivity:Payment for request ID b903d749cd814e099f06ebf4a56a2f90 processed successfully
+INFO:UpdateInventoryActivity:Checking inventory for order b903d749cd814e099f06ebf4a56a2f90 for 11 cars
+INFO:UpdateInventoryActivity:There are now 89 cars left in stock
+INFO:NotifyActivity:Order b903d749cd814e099f06ebf4a56a2f90 has completed!
+Workflow completed! Result: COMPLETED
+  Purchase of item is  COMPLETED
+```
+
+### View workflow output with Zipkin
+
+For a more detailed view of the workflow activities (duration, progress etc.), try using Zipkin.
+
+1. Launch Zipkin container - The [openzipkin/zipkin](https://hub.docker.com/r/openzipkin/zipkin/) docker container is launched on running `dapr init`. Check to make sure the container is running. If it's not, launch the Zipkin docker container with the following command.
+
+```bash
+docker run -d -p 9411:9411 openzipkin/zipkin
+```
+
```

Comment:
I've done net-new dapr deployments where I had to do this. I don't remember the circumstances but it probably doesn't hurt to mention it for users since it only improves usability. 

That said, I'm  fine either way since I can't remember why I've had to do this in the past.

Reply:
My vote is we remove.  The default of the CLI it is there and running.

Reply:
I have removed the phrase and command to run zipkin container. It is enabled by default with `dapr init`.
Feel free to add it back in case of any concerns.